envs/environment_py3.6_iris2.2.yml

```diff
@@ -21,8 +21,6 @@ dependencies:
   - pysteps
   - isort=4.3.21
   - black=19.10b0
-  - bandit
```

Comment:
Bandit and safety are being removed from this environment, but they are used in tests.yml.
Suggest not removing these libraries as the bandit/pytest job in tests.yml is currently run in both environments.

Reply:
I've fixed this.